### PR TITLE
Installer refactoring

### PR DESCRIPTION
**Pull request by jdsn**
small refactoring, needed for the next PR to support the new installer

### CODE REVIEW

**Comment by bmwiedemann** [comment]
:+1:

**Comment by rsalevsky** [comment]
+1

**Comment by aplanas** [comment]
The gating fails launching the crowbar service:

```
13:53:11 + service crowbar status
13:53:11 crowbar.service - Crowbar
13:53:11    Loaded: loaded (/etc/systemd/system/crowbar.service; disabled)
13:53:11    Active: inactive (dead)
```

**Comment by jdsn** [comment]
> The gating fails launching the crowbar service:

Nope. It is actually running. The code does a 'stop', 'status', 'start'.
What you cited was the status of the stopped service.

It is running but the self test fails.

File: scripts/mkcloud [hunk 1]
```diff
@@ -594,7 +596,7 @@ function instcrowbar()
     scp_install_chef_suse_override
     onadmin installcrowbar
     local ret=$?
-    $scp root@$adminip:screenlog.0 "$artifacts_dir/screenlog.0.install-suse-cloud"
+    $scp root@$adminip:$crowbar_install_log "$artifacts_dir/"
```
Comment (tboerger) [inline scripts/mkcloud]: This variable seems to be defined in `qa_crowbarsetup.sh` only.

File: scripts/mkcloud [hunk 2]
```diff
@@ -604,7 +606,7 @@ function instcrowbarfromgit()
     safely rsync -av --exclude ".git" --exclude ".ci-tracking" ./crowbar root@$adminip:/root/
     onadmin installcrowbarfromgit
     local ret=$?
-    $scp root@$adminip:screenlog.0 "$artifacts_dir/screenlog.0.install-suse-cloud"
+    $scp root@$adminip:$crowbar_install_log "$artifacts_dir/"
```
Comment (tboerger) [inline scripts/mkcloud]: This variable seems to be defined in `qa_crowbarsetup.sh` only.

File: scripts/mkcloud (hunk 1 above)
Comment (MaximilianMeister) [inline scripts/mkcloud]: `qa_crowbarsetup.sh` gets sourced by before mkcloud, so that's fine

**Comment by MaximilianMeister** [comment]
:+1:

**Comment by jdsn** [comment]
> The problem is a timeout launching crowbar.

Nope. Crowbar was launched, but I accidentally removed the sleep 20 following the start command. So the check that started right after the start command could not contact crowbar and let the run fail.
With only the sleep added back, the build passed.

**Pull request merged**